numpy/core/src/private/ufunc_override.c

```diff
@@ -134,6 +134,14 @@ PyUFunc_WithOverride(PyObject *args, PyObject *kwds,
                 goto fail;
             }
             if (with_override != NULL) {
+                for (j = 0; j < num_override_args; j++) {
```

Comment:
The `int j` can go on the line above this - there's no need to introduce it 70 lines earlier